src/db/schema.ts

```diff
@@ -0,0 +1,21 @@
+import { decimal, pgEnum, pgTable, serial, varchar } from "drizzle-orm/pg-core";
+
+export const genreEnum = pgEnum("genre", ["punk", "rock", "indie"]);
```

Comment:
the second argument should probably be a contant array, so you can actually reuse somewhere, without needing to access at through `genreEnum.enumValues`,  but this is I guess personal values. But you would most likely want to use a variables names `GENRES` and put it somewhere into a constants file.